### PR TITLE
I18next setup

### PR DESCRIPTION
**Pull request by MiraAr**
PTAL

### CODE REVIEW

**Review by emrerdem1 — APPROVED** [review]

File: src/containers/App.test.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ import React from "react";
 import { render } from "@testing-library/react";
 import App from "./App";
 
-test("renders learn react link", () => {
+test("renders correctly whithout crash", () => {
```
Comment (louisrli) [inline src/containers/App.test.js]: i think you can just delete this file, we don't want random code like this lying around

File: yarn.lock [hunk 2]
```diff
@@ -5930,6 +5934,16 @@ html-minifier-terser@^5.0.1:
     relateurl "^0.2.7"
     terser "^4.6.3"
 
+<<<<<<< HEAD
```
Comment (louisrli) [inline yarn.lock]: merge conflicts here

File: src/components/home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import { useTranslation } from "react-i18next";
+
+export default function Home() {
+  const { t } = useTranslation();
+
+  function setLanguage(language) {
+    i18next.changeLanguage(language);
+  }
+
+  return (
```
Comment (louisrli) [inline src/components/home/Home.jsx]: even for basic setup you should move this out into a different component, such as languagedropdown component. eventually you have to move it out anyway, so why not do it now :)

**Comment by MiraAr** [comment]
PTAL

File: src/components/home/Home.test.jsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Home from "./Home";
+it("renders correctly", () => {
```
Comment (louisrli) [inline src/components/home/Home.test.jsx]: we actually don't need a home.test.jsx usually we test the individual components only

**Review by louisrli — APPROVED** [review]
delete home.test.jsx and the snapshot and otherwise its good, get another approving review

**Review by OmarAlithawi — APPROVED** [review]

**Pull request merged**